apps/nextjs/src/app/api/chat/webActionsPlugin.ts

```diff
@@ -56,17 +57,24 @@ export const createWebActionsPlugin: PluginCreator = (
       throw new Error("User ID not set");
     }
 
-    await inngest.send({
-      name: "app/slack.notifyModeration",
-      user: {
-        id: userId,
-      },
-      data: {
-        chatId: aila.chatId || "Unknown",
-        categories: moderation.categories as string[],
-        justification: moderation.justification ?? "Unknown",
-      },
-    });
+    try {
+      log.info("Sending slack notification");
+      await inngest.send({
+        name: "app/slack.notifyModeration",
+        user: {
+          id: userId,
+        },
+        data: {
+          chatId: aila.chatId || "Unknown",
+          categories: moderation.categories as string[],
+          justification: moderation.justification ?? "Unknown",
+        },
+      });
+    } catch (e) {
+      log.error("Error sending slack notification", e);
```

Comment:
This is async so "error scheduling slack..."?

Reply:
Good call, thanks